htdocs/compta/stats/index.php

```diff
@@ -461,22 +462,22 @@
 			//var_dump($annee.' '.$year_end.' '.$mois.' '.$month_end);
 			if ($annee < $year_end || ($annee == $year_end && $mois <= $month_end)) {
 				if ($annee_decalage > $minyear && $case <= $casenow) {
-					if (!empty($cum[$caseprev]) && !empty($cum[$case])) {
-						$percent = (round(($cum[$case] - $cum[$caseprev]) / $cum[$caseprev], 4) * 100);
-						//print "X $cum[$case] - $cum[$caseprev] - $cum[$caseprev] - $percent X";
+					if (!empty($cum_ht[$caseprev]) && !empty($cum_ht[$case])) {
```

Comment:
@rycks can you rename cum to cumulative 🙄

Reply:
@frederic34 for sure i could do it but i don't know why ? could you please explain it to me (in private mail if that's something embarrassing)

Reply:
see translation of cum... in french

Reply:
and ... that's a remake of weboob story ?

excuse-me but if we have to change all variables names that could be misinterpreted we could have a lot of work "for nothing" ...

Reply:
Le me do an other PR for that